src/coreclr/jit/unwind.h

```diff
@@ -10,7 +10,8 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 */
 
-#ifdef TARGET_ARMARCH
+////TODO for LOONGARCH64: should seperately define for loongarch64.
```

Comment:
Where is the LoongArch64 unwind codes specification available?

Is it just a copy of the arm64 unwind codes?

E.g., unwindSaveRegPair() is not used; should it be removed? DumpUnwindInfo() looks like a copy with lots of things `#ifdef'ed` out.

Reply:
> Is it just a copy of the arm64 unwind codes?

yes, a copy from arm64.

> E.g., unwindSaveRegPair() is not used; should it be removed? DumpUnwindInfo() looks like a copy with lots of things `#ifdef'ed` out.

yes, some interfaces are not used by LoongArch64.